### DISCUSSION TITLE
RadioGroup form validation

### DISCUSSION THREAD

**Discussion by psebestyen** [rsuite/rsuite#3373] [Q&A]
I am trying to validate that one of the radio buttons is checked within a radio group when the form is submitted.
```
<RadioGroup>
  <Radio name="red" value="red">Red</Radio>
  <Radio name="green" value="green">Green</Radio>
</RadioGroup>
```
Of course I would like some error message to be displayed as well. How would I do that with the built in Schema validator?

Thanks

**Comment by SevenOutman** [comment]
Hi @psebestyen, when you use RadioGroup for selecting one option among several, the `name` attribute should be set on RadioGroup rather than Radio.

```jsx
<RadioGroup name="color">
  <Radio value="red">Red</Radio>
  <Radio value="green">Green</Radio>
</RadioGroup>
```

When one of the radios is checked, the `color` field of the form will be set to its value, what you need to do is to validate that `color` field is filled with a known option.
